### PR TITLE
[Unity] User Profile & Version Update

### PR DESCRIPTION
**Pull request by AliRKat**

### CODE REVIEW

**Comment by AliRKat** [comment]
[[Unity, Doc] User Profile Update](https://trello.com/c/alvxl7hQ/1939-unity-doc-user-profile-update)

File: unity/next.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p>
-  This documentation is for the Countly Unity SDK version 23.12.X. The SDK source
+  This documentation is for the Countly Unity SDK version 24.8.X. The SDK source
```
Comment (arifBurakDemiray) [inline unity/next.md:2]: not just user profile update, plus version

**Review by arifBurakDemiray — APPROVED** [review]

File: unity/next.md [hunk 2]
```diff
@@ -979,139 +979,245 @@ Countly.Instance.Init(config);</code></pre>
...
-    <strong>Gender</strong>: gender of the user (use only single char like ‘M’
-    for Male and ‘F’ for Female).
-  </li>
-  <li>
-    <strong>BirthYear</strong>: birth year of the user.
-  </li>
-</ul>
+<pre><code class="!whitespace-pre hljs language-csharp">// Provide multiple properties at once within a dictionary
+Countly.Instance.UserProfile.SetProperties(Dictionary &lt;string, object&gt; userProperties);
+<br>// Provide single user property as key and value
```
Comment (arifBurakDemiray) [inline unity/next.md:1003]: dont use br bro, just enter

Comment (AliRKat) [reply unity/next.md:1003]: sometimes zendesk adds those when i enter. i am no maniac who manually adds br in there :D

File: unity/next.md (hunk 1 above)
Comment (AliRKat) [reply unity/next.md:2]: renamed PR

File: unity/next.md (hunk 2 above)
Comment (arifBurakDemiray) [reply unity/next.md:1003]: :hulk:

File: unity/next.md (hunk 1 above)
Comment (arifBurakDemiray) [reply unity/next.md:2]: you are here ❤️

File: unity/next.md [hunk 3]
```diff
@@ -979,139 +979,245 @@ Countly.Instance.Init(config);</code></pre>
   If consent is required, recording Star Rating requires<code>StarRating</code><span>consent. If consent is required and not given, recording a Star Rating will not be possible.</span><span></span>
 </p>
 <h1 id="h_01HABTZ3164RBD3AC31PH330W9">User Profiles</h1>
+<div class="callout callout--info">
+  <p>User Profiles is an enterprise-level plugin.</p>
```
Comment (turtledreams) [inline unity/next.md:983]: lets change this ali

**Review by turtledreams — CHANGES_REQUESTED** [review]

**Review by peterBrxwn — APPROVED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**